test/lib/usd/translators/testUsdExportMesh.py

```diff
@@ -45,6 +45,7 @@ def _AssertVec3fArrayAlmostEqual(self, arr1, arr2):
                 self.assertAlmostEqual(arr1[i][j], arr2[i][j], places=3)
 
     def testExportAsCatmullClark(self):
+        cmds.file(self.meshTestFile, force=True, open=True)
```

Comment:
Why did you have to make this change?

Reply:
Otherwise it was opening up the file at the setup of the class, and AFAIK running of the tests is non deterministic. So if my test ran in between an existing test and the older ones, I'd have broken them. This made it slower, but safer.